crates/trie/common/src/prefix_set.rs

```diff
@@ -96,6 +96,15 @@ where
     }
 }
 
+impl Extend<Nibbles> for PrefixSetMut {
+    fn extend<T>(&mut self, iter: T)
+    where
+        T: IntoIterator<Item = Nibbles>,
+    {
+        self.keys.extend(iter);
```

Comment:
this doesn't account for `self.all` and potentially conflicts with `PrefixSetMut::extend`